src/user-utilities.js

```diff
@@ -50,7 +50,21 @@ function createUser(firstName, lastName, email, phoneNumber) {
 
 }
 
+function   manageUsers(user, a) {
+  if ( a== "delete"){
+    return undefined
+} else {
+      if  (a != "delete" && a == "duplicate") {
```

Comment:
could you explain why this behavior is this way?
Its a bit confusing for it to do delete and duplicate

Reply:
Can we hop on a call so I can walk through it?